cg/models/demultiplex/flow_cell.py

```diff
@@ -126,6 +126,21 @@ def sequencer_type(
         """Return the sequencer type."""
         return sequencer_types[self.machine_name]
 
+    def get_bcl_converter(self, bcl_converter: str) -> str:
```

Comment:
I don't like that we still pass the `bcl_converter` as an argument - it is confusing and I think it is cleaner if we just rely on retrieving it based on the sequencing machine type.

Maybe this involves more work to ensure only `BclConverter.BCLCONVERT` enum is used and not `BclConverter.DRAGEN` as to not introduce bugs. I guess that might explain why you chose to keep the `bcl_converter` argument - but it should be removed. Might have to be in a separate PR, you decide :)

Reply:
I am unsure if we would still want to manually demultiplex a novaseq flowcell with Bcl2fastq. If that is the case, `bcl_converter` would still be needed. I left that possibility open when writing the sample sheet code, but I am not sure how often we use this and if it is needed. I agree that we should remove it if we don't need it.

Reply:
explained above. Reproducability reasons if those exist. Need input from Karl on this.